packages/schematics/src/collection/node-application/files/nestjs/main.ts__tmpl__

```diff
@@ -0,0 +1,11 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app/app.module';
+
+const port = 3333;
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule, { logger: false });
```

Comment:
`{ logger: false }` should be removed, but I need help to get the e2e test working properly with the default logger enabled, due to multiple lines being printed.